tools/packaging/kernel/configs/fragments/common/base.conf

```diff
@@ -4,13 +4,6 @@ CONFIG_SECTION_MISMATCH_WARN_ONLY=y
 CONFIG_SMP=y
 # Note, no nested VM support enabled here
 
-# Turn off embedded mode, as it disabled 'too much', and we
-# no longer pass all the tests. We should refine this, and
-# work out which of the ~66 items it enables are really needed.
-# I believe this is the actual syntax we need for a fragment to
-# disable an item...
-# CONFIG_EMBEDDED is not set
```

Comment:
I guess it is fine to not mention CONFIG_EMBEDDED in the changelog since it wasn't set.

Reply:
Note that this was removed in kernel 6.6, not 6.5.